deny.toml

```diff
@@ -98,7 +99,7 @@ confidence-threshold = 0.8
 # Allow 1 or more licenses on a per-crate basis, so that particular licenses
 # aren't accepted for every possible crate as with the normal allow list
 exceptions = [
-  { allow = ["GPL-2.0"], name = "mysqlclient-src" },
```

Comment:
Is `mysqlclient-src` not in our dependency tree anymore?

Reply:
No, it is. Honestly I am confused about this as well, but cargo deny stopped complaining about this license and started warning about this exception.